pyspark/bigdl/contrib/onnx2bigdl/load_graph.py

```diff
@@ -0,0 +1,131 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+from bigdl.nn.onnx.layer import *
+from bigdl.nn.layer import Identity, Model
+from .ops_mapping import _convert_map as convert_map
+
+
+class OnnxGraph(object):
+	def __init__(self, graph_proto=None):
+		self._inputs = list()
+		self._outputs = list()
+		self._tensors = dict()
+		self._modules = dict()
+		self._root = list()
+		self._dummy_root = Identity()()
+		self._graph = self.load_graph(graph_proto)
```

Comment:
why you need this private member ? I did not see its usage.  And `load_graph` will be called on initialization, why do we still call it outside ? it's duplicated.

Reply:
Thanks, will remove it.